### DISCUSSION TITLE
Do you think I was blocked by Google?

### DISCUSSION THREAD

**Discussion by kernelcrunch** [tmiland/invidious-updater#76] [General]
My new site has been working fairly well for the past week or two.  I've been ironing out the kinks, such as scheduled restarts, etc.  Suddenly, yesterday, none of my images will download.  The page seems to load but without images/videos embedded.  Even if I restore a VM snapshot from a week ago when I'm sure it was working, same thing.  My best guess is that Google is blocking me.  I added the BadBots block lists to my firewall, but maybe I missed something.  Anyway, that's the best guess of what happened because I simply cannot get it to work anymore.  I thought I'd mention it here to see if you had any thoughts?  Is this what it looks like when Google blacklists an IP?  Or, could I be way off and it's just something else going on.  Thanks again!

**Comment by tmiland** [comment]
Hello,

please have a look here: https://github.com/iv-org/invidious/issues/2577

Do a update and turn off/remove the `use_quic` option in the config file, hopefully it will resolve the issue.

**Comment by kernelcrunch** [comment]
Bingo!  Thanks so much!  Everything is working again. I feel a little silly for not finding that on my own.

**Reply by tmiland** [reply]
Great!

My pleasure! 👍
